cmd/root.go

```diff
@@ -60,38 +68,89 @@ func initConfig() {
 	}
 }
 
+var (
+	eventsCounter = prometheus.NewCounter(prometheus.CounterOpts{
+		Namespace: "ssl",
```

Comment:
What about:
namespace -> ssl_agent / http_agent
SubSystem -> queue / api
name .... as this